src/safe_cli/safe_creator.py

```diff
@@ -206,22 +226,31 @@ def main(*args, **kwargs) -> EthereumTxSent:
         )
 
         proxy_factory = ProxyFactory(proxy_factory_address, ethereum_client)
-        expected_safe_address = proxy_factory.calculate_proxy_address(
-            safe_contract_address, safe_creation_tx_data, salt_nonce
-        )
-        if ethereum_client.is_contract(expected_safe_address):
-            print_formatted_text(f"Safe on {expected_safe_address} is already deployed")
-            sys.exit(1)
-
-        if yes_or_no_question(
-            f"Safe will be deployed on {expected_safe_address}, looks good?"
-        ):
-            ethereum_tx_sent = proxy_factory.deploy_proxy_contract_with_nonce(
-                account, safe_contract_address, safe_creation_tx_data, salt_nonce
+        if generate_vanity_addresses:
+            for vanity_salt_nonce in range(2**256):
+                expected_safe_address = proxy_factory.calculate_proxy_address(
+                    safe_contract_address, safe_creation_tx_data, vanity_salt_nonce
+                )
+                print_formatted_text(f"{expected_safe_address} {vanity_salt_nonce}")
+        else:
+            expected_safe_address = proxy_factory.calculate_proxy_address(
+                safe_contract_address, safe_creation_tx_data, salt_nonce
             )
-            print_formatted_text(
-                f"Sent tx with tx-hash={ethereum_tx_sent.tx_hash.hex()} "
-                f"Safe={ethereum_tx_sent.contract_address} is being created"
-            )
-            print_formatted_text(f"Tx parameters={ethereum_tx_sent.tx}")
-            return ethereum_tx_sent
+            if ethereum_client.is_contract(expected_safe_address):
+                print_formatted_text(
+                    f"Safe on {expected_safe_address} is already deployed"
+                )
+                sys.exit(1)
+
+            if yes_or_no(
+                f"Safe will be deployed on {expected_safe_address}, looks good?"
```

Comment:
```suggestion
                f"Safe will be deployed on {expected_safe_address}. Would you like to proceed?"
```

Reply:
This has been here forever, but looks good 😉

Reply:
Done